src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,357 @@
+
+All of the modules above, aside from yamux, are mandatory.
+
+One notable omission is the [libp2p Kademlia DHT](https://github.com/libp2p/specs/tree/master/kad-dht); The networking specification instead specifies the use of discv5.
+The rationale is [provided here](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#why-are-we-using-discv5-and-not-libp2p-kademlia-dht).
+
+The specific design details are out of scope for this post but are [described in great detail here](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#design-decision-rationale).
+Please check it out to see why TCP was chosen (and not QUIC), why Noise, why GossipSub, etc.
+
+### Network Statistics 📊
+
+The number of networked Beacon Chain nodes (i.e. nodes running libp2p) is [around 5,000 at the current count](https://migalabs.es/beaconnodes) with this [client diversity](https://clientdiversity.org/#distribution).
+
+<div class="container" style="display:flex; column-gap:10px;">
+    <figure>
+        <img src="../assets/eth-libp2p-beacon-nodes.png" width="600">
+        <figcaption style="font-size:x-small;">
+            <a href="https://migalabs.es/beaconnodes">[Miga Labs dashboard.]</a>
+        </figcaption>
+    </figure>
+</div>
+
+libp2p is running on all those consensus clients; a fantastic testament to 6+ years of hard work!
+
+Additionally, there are over [495,000 validators](https://beaconcha.in/validators) amongst these nodes (validators are [virtual entities](https://launchpad.ethereum.org/en/faq) that live on the Beacon Chain.)
+
+## What's next? :rocket:
+
+Now that the Merge has gone smoothly, the primary collaboration efforts have been resolved successfully.
+At this time, Protocol Labs is supporting the Ethereum community in ensuring the libp2p is running smoothly and reliably.
+Some of the future roadmap items for libp2p and Ethereum have been hinted at in the specification (like exploring the usage of QUIC).
+When the time comes for that, the collaboration will pick up steam once more!
+
+In conclusion, the libp2p and Ethereum integration has been a massive effort.
+It could not have been possible without the grit and perseverence of many organizations and invididuals.
+Thank you to everyone invovled, and onwards to our next goals!
+
+We hope you enjoyed our summary of the journey and hope you learned a few things along the way.
+Thank you for reading! :pray:
+
+### Resources and how you can help contribute 💪
+
+If you would like to learn more about libp2p, please see the libp2p:
+
+- [Docs](https://docs.libp2p.io/)
+- [Connectivity Website](https://connectivity.libp2p.io/)
+    - This describes the various libp2p transport implementations
+- [libp2p Specifications](https://github.com/libp2p/specs/)
+
+If you would like to contribute, please [connect with the libp2p maintainers](https://libp2p.io/#community).
+
+---
+## Comparing devp2p and libp2p 🤼
```

Comment:
What about putting this as a page under https://docs.libp2p.io/concepts

Basically I think it would be useful for us to be transparent/up front about alternatives to libp2p including devp2p.  It's fine if we just have devp2p for now and we can put this content int here.  We can also link from there to the blog entry to learn more of the history.

This blog entry can then link to that content.

Reply:
That's a great idea, I'll update the docs site to add a page on comparing libp2p to other stacks and link out to it from here.

Reply:
Done